internal/receiver/smartagentreceiver/output.go

```diff
@@ -89,7 +89,7 @@ func getMetadataExporters(config Config, host component.Host, nextConsumer consu
 	}
 
 	if len(exporters) == 0 {
-		logger.Warn("no dimension updates are possible as no valid metadataClients have been provided and next pipeline component isn't a MetadataExporter")
```

Comment:
@rmfitzpatrick seems like this results in a warning for any receiver that does not use metadata updates. I changed the logging level to `Info`. But do we really need to log this out?

Reply:
Good call, debug makes sense to me